duckduckgo/iOS
# Sabrina/sync setup update

Task/Issue URL: https://app.asana.com/0/0/1205948195318715/f

**Description**: Update the sync UI flow according to https://www.figma.com/file/5wIXwQLKJ0dZVsP5XWcgbT/%F0%9F%8E%AF-Sync-9?type=design&node-id=8748-306047&mode=design&t=5IygRcQcvvNAJ7xg-0



**Steps to test this PR**:
1. Go to Sync Settings check UI and copy looks as expected

Sync This Device Flow
1. Go to Sync Settings check UI and copy looks as expected
3. Click on Sync and Back Up this device check “Sync and Back Up this device” sheet appears and design and copy are as expected
4. Click on “Turn on Sync & Backup”
5. Check preparing to sync sheet appears briefly before Save Recovery Code appears (check copy and UI)
6. Check save as PDF and copy code work as expected
7. Press Next and check Your Data is Synced! sheet appears.
8. Press done and check sync enabled view appears (check looks as expected)
9. Delete account

Recover Synced Data flow
1. Click on Recover Synced Data and check Recover Synced Data sheets appear and looks as expected
2. Click on Get Started and check the recover synced data camera mode appears 
3. Try to scan a recovery QR code or other device code and check the flow appears as in the previous case
4. Try the same with Enter text Code manually

Sync with Another device flow
1.  Click on Sync with Another Device
2. Check the Scan QR code camera view appears
3.  Try to point to a QR code of both a connected device and later an unconnected device and check the flow
4. Same by manually inserting the code
5. Same when scanning the QR code presented from another device
6. Same when copying the text in another device.

Sync enabled View
Check sync with another device work as expected: the same ScanQR code view should appear and you should be able to connect a new device.

<!—
Before submitting a PR, please ensure you have tested the combinations you expect the reviewer to test, then delete configurations you *know* do not need explicit testing.

Using a simulator where a physical device is unavailable is acceptable.
—>

**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


—
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
